src/Dispatcher.php

```diff
@@ -11,6 +11,7 @@
 use Penny\Exception\MethodNotAllowed;
 use Penny\Exception\RouteNotFound;
 use Psr\Http\Message\RequestInterface;
+use Penny\Route\RouteInfo;
 
 class Dispatcher
```

Comment:
don't know effectively is a wrapper around `FastRouterDispatcherInterface` interface in order... Maybe we can let this as Dispatcher for now...